test/fixtures/terms.ttl

```diff
@@ -0,0 +1,22 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+<https://example.com/resources/artwork>
+    a skos:Concept ;
+    skos:prefLabel "Nachtwacht" ;
+    skos:altLabel "Nachtwacht alt" ;
+    skos:scopeNote "One of the most famous Dutch paintings" ;
+    skos:hiddenLabel "Hidden label" ;
+    skos:broader <https://example.com/resources/2> ;
+    skos:ignored "Not used by Network of Terms" .
+
+<https://example.com/resources/painting>
+    a skos:Concept ;
+    skos:prefLabel "Painting" ;
+    skos:related <https://example.com/resources/art> ;
+    skos:broader <https://example.com/resources/art> .
+
+<https://example.com/resources/art>
+    a skos:Concept ;
+    skos:prefLabel "Art" ;
+    skos:altLabel "All things art" ;
+    skos:narrower <https://example.com/resources/painting> .
```

Comment:
Does this test data make enough sense so it won’t hinder maintenance in the future?